.changeset/rotten-rockets-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-surveycto': patch
+---
+
+Fix servername typo
```

Comment:
Can you just qualify that this is in the configuration schema? And for facebook

Reply:
Done ✅

Reply:
Done ✅